### ISSUE TITLE
[FR] webhook POST request handling

### ISSUE THREAD

**Issue by ITwrx** [iffy/nim-stripe#2]
If there were a proc(s) and example for handling a webhook POST request (signature verification+exception and accessing the posted data), I would think it would make this library pretty full featured/flexible for how small it is.

Thanks

**Comment by iffy** [comment]
I'd be up for that functionality being in this library, though I've never used Stripe webhooks myself so I'm not sure I'd be the best person to implement it.

**Comment by ITwrx** [comment]
fair enough. :) i've only used webhooks via the php library. If i get it working in nim i'll update this issue.

**Comment by ITwrx** [comment]
just for reference: https://stripe.com/docs/webhooks/signatures#verify-manually
